### DISCUSSION TITLE
Is Parcel dead? ☠️

### DISCUSSION THREAD

**Discussion by ftzi** [parcel-bundler/parcel#9790] [Q&A] [answered]
It's not getting updates anymore.

Should we move to vite or webpack?

**Comment by devongovett** [comment, marked as answer]
Not dead - you can see the commits occurring pretty much daily: https://github.com/parcel-bundler/parcel/commits/v2/ Atlassian has a team of several full time engineers working on it.

We're currently working on porting more of Parcel to Rust, which is a large project that will require some time to complete. So releases may be slower temporarily.

**Reply by ftzi** [reply]
Thanks! For me it was being odd that commits are happening for quite a while but no new versions are being released.

**Reply by adamhenson** [reply]
Hey @devongovett any chance a patch version could be published in the near future from a sha that predates the Parcel to Rust work? My team has been suffering for a little while from issues that we think is fixed by #9640. I didn't realize until today Parcel publishes nightly releases and the issue seems fixed from my only recent testing in release `2.0.0-canary.1665`. I had difficulty tracking down this release - I take it the canary releases are published nightly from the `v2` branch (as in `2.0.0-canary.1665` is newer than `2.12.0`)?

**Reply by mischnic** [reply]
The current Parcel-to-Rust work should not have affected the existing implementation, so even the current canaries should still work just fine (and be backwards compatible), so `yarn add parcel@canary`
That said, `parcel@2.0.0-canary.1557` was published just after that PR as part of commit 2df56ed286ae41610d5ed9d439c37d6cd02c8ead .

**Reply by adamhenson** [reply]
Thanks @mischnic - cool - ya, you're right - we could probably use the canary version 👍

**Reply by fregante** [reply]
Would it be possible to release what's on main at the moment? It contains a bug fix that has been blocking me for half a year: https://github.com/parcel-bundler/parcel/issues/9458 and now Chrome is forcing me to update and essentially leave Firefox users in the dust without this fix. 

Also see this Parcel-specific [StackOverflow question](https://stackoverflow.com/q/75043889) where the user is also unable to use Parcel to apply the solution

**Reply by adamhenson** [reply]
@fregante I think @mischnic's answer to [my question](https://github.com/parcel-bundler/parcel/discussions/9790#discussioncomment-9906405) would apply to yours also. [The canary versions](https://www.npmjs.com/package/parcel?activeTab=versions) are published from the main branch apparently (`2.0.0-canary.*`).

**Reply by fregante** [reply]
I've had trouble matching the version across plugins in the past that way, but I'll give it a try again

**Reply by adamhenson** [reply]
For me and my team, using a canary version will work for now, but it was a hard sell for me as we'd have more confidence in using a "stable" version. Since there hasn't been a new stable Parcel release since February (2024), it would be great to have a new patch / minor / or even major version release... and I think it would give the community better faith in the project. Just my 2 cents for what it's worth. Definitely appreciate this project and appreciate having the canary release option.
